packages/pancake-svg/src/svg.js

```diff
@@ -0,0 +1,122 @@
+/***************************************************************************************************************************************************************
+ *
+ * Generate and compile SVG's
+ *
+ * @repo    - https://github.com/govau/pancake
+ * @author  - Dominik Wilkowski
```

Comment:
Surely I'm not the author :)

Reply:
if your happy for my name to go on it, so be it :)

Reply:
yeah well you are the author of this plugin ;)